spec/datadog/profiling/collectors/cpu_and_wall_time_worker_spec.rb

```diff
@@ -199,7 +200,9 @@
 
         exception = try_wait_until(backoff: 0.01) { cpu_and_wall_time_worker.send(:failure_exception) }
 
-        expect(exception.message).to include "pre-existing SIGPROF"
+        expect(exception.message).to include "(empty_signal_handler): There's a pre-existing SIGPROF"
+        # The handler name and signal name are statically check as safe for telemetry
+        expect(exception.telemetry_message).to include "(empty_signal_handler): There's a pre-existing SIGPROF"
       end
```

Comment:
If possible, I'd like to avoid the additional testing of the telemetry message -- the `telemetry/logging_spec.rb` should be the one responsible for testing that.

(This applies to other specs below)